packages/@magic-sdk/provider/test/spec/modules/auth/loginWithCredential.spec.ts

```diff
@@ -34,30 +36,37 @@ test('If no parameter is given & platform target is "web", URL search string is
 
   await magic.auth.loginWithCredential();
 
-  const requestPayload = magic.user.request.mock.calls[0][0];
+  const requestPayload = magic.auth.request.mock.calls[0][0];
   expect(requestPayload.jsonrpc).toBe('2.0');
   expect(requestPayload.method).toBe('magic_auth_login_with_credential');
   expect(requestPayload.params).toEqual(['?magic_credential=asdf']);
 });
 
 test('If no parameter is given & platform target is NOT "web", credential is empty string', async () => {
   const magic = createMagicSDK({ platform: 'react-native' });
+  magic.auth.request = jest.fn();
 
   await magic.auth.loginWithCredential();
 
-  const requestPayload = magic.user.request.mock.calls[0][0];
+  const requestPayload = magic.auth.request.mock.calls[0][0];
   expect(requestPayload.jsonrpc).toBe('2.0');
   expect(requestPayload.method).toBe('magic_auth_login_with_credential');
   expect(requestPayload.params).toEqual(['']);
 });
 
 test('If `testMode` is enabled, testing-specific RPC method is used', async () => {
   const magic = createMagicSDKTestMode({ platform: 'web' });
+  magic.auth.request = jest.fn();
 
   await magic.auth.loginWithCredential('helloworld');
 
-  const requestPayload = magic.user.request.mock.calls[0][0];
+  const requestPayload = magic.auth.request.mock.calls[0][0];
   expect(requestPayload.jsonrpc).toBe('2.0');
   expect(requestPayload.method).toBe('magic_auth_login_with_credential_testing_mode');
   expect(requestPayload.params).toEqual(['helloworld']);
 });
+
+test('method should be a promi event', () => {
```

Comment:
Super tiny nit: this test case could probably have a better name like `method return result should be a PromiEvent` or something like that.

Reply:
Makes sense. I'll update.